src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -236,14 +246,36 @@ public PlayerMock addPlayer()
 	 * @param name The name to give to the player.
 	 * @return The added player.
 	 */
-	public PlayerMock addPlayer(String name)
+	@NotNull
+	public PlayerMock addPlayer(@NotNull String name)
 	{
 		assertMainThread();
 		PlayerMock player = new PlayerMock(this, name);
 		addPlayer(player);
 		return player;
 	}
 
+	/**
+	 * Removes previously added player, calls PlayerQuitEvent on server
+	 *
+	 * @param player player to remove.
+	 * @return PlayerQuitEvent that was called on a server.
+	 */
+	@NotNull
+	public PlayerQuitEvent removePlayer(@NotNull PlayerMock player)
```

Comment:
Not sure if a `remove` operation should return an event here...

Reply:
If someone would want to getExitMessage he would need to get this event I think

Reply:
So I'm not sure what ServerMock.removePlayer should return here, so I would change this to void and make another function called ServerMock.getLastPlayerQuitMessage() if someone would need it.

Reply:
I've actually made ServerMock.getLastPlayerQuitEvent() in case when Bukkit devs would add some other data to PlayerQuitEvent that someone would want to retrieve